crypto/keys/hd/hdpath_test.go

```diff
@@ -168,16 +169,15 @@ func ExampleSomeBIP32TestVecs() {
 	// bip32 path: m/0/7
 	seed = mnemonicToSeed("monitor flock loyal sick object grunt duty ride develop assault harsh history")
 	master, ch = ComputeMastersFromSeed(seed)
-	priv, _ = DerivePrivateKeyForPath(master, ch, "0/7")
+	priv, err = DerivePrivateKeyForPath(master, ch, "m/0/7")
+	require.NoError(t, err) // TODO: shouldn't this error?
 	fmt.Println(hex.EncodeToString(priv[:]))
 
 	// Output: keys from fundraiser test-vector (cosmos, bitcoin, ether)
 	//
 	// bfcb217c058d8bbafd5e186eae936106ca3e943889b0b4a093ae13822fd3170c
 	// e77c3de76965ad89997451de97b95bb65ede23a6bf185a55d80363d92ee37c3d
 	// 7fc4d8a8146dea344ba04c593517d3f377fa6cded36cd55aee0a0bb968e651bc
-	// INVALID
-	// INVALID
 	//
```

Comment:
@fedekunze this was a testable example, you turned it into a standard test case (`ExampleSomeBIP32TestVecs()` -> `TestBIP32Vecs(t *testing.T)`). Mind elaborating why?

Reply:
what's the purpose of a testable example?